README.md

```diff
@@ -1,15 +1,15 @@
 # AMP Packager
 
-AMP Packager is a tool to [improve AMP
-URLs](https://www.ampproject.org/latest/blog/developer-preview-of-better-amp-urls-in-google-search).
+AMP Packager is a tool to [Serve AMP using Signed Exchanges](https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/signed-exchange/) which in turn [improves AMP
+URLs](https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/signed-exchange/).
```

Comment:
I like leading with the purpose, the intended effect of fixing AMP URLs, and following with the means (signed exchanges). I'd also lean towards not having two links to the same URL in the same sentence; the user will probably click both not realizing they're the same (or at least hover, costing time). So, e.g.

```
AMP Packager is a tool to [improve AMP URLs](https://amp.dev/documentation/guides-and-tutorials/optimize-and-measure/signed-exchange/)
by serving AMP using signed exchanges.
```

FWIW the second para goes into detail on signed exchanges, so the mention might not be necessary in the first para, but I'm okay with it either way.

Reply:
Oops. I meant to change the 2nd link to the correct blog post, rather than have them both be the same. If you'd prefer, I can just remove it.